### DISCUSSION TITLE
primevue icons disappeared.

### DISCUSSION THREAD

**Discussion by Uzeyir9695** [primefaces/community#1362] [PrimeVue]
After 3.49.1 version primevue icons disappeared.

**Comment by tugcekucukoglu** [comment]
How can we replicate the problem?

**Reply by Uzeyir9695** [reply]
on my local when I run `npm run dev` (I am using laravel / Vue stack) icons is not recognized, but once I run `npm run build` and stop `npm run dev`, icons appears. please view the screenshot attached below. **`Book session`** button has `pi pi-check` icon on the left side but how it looks.
![iconnot](https://github.com/primefaces/community/assets/74812496/4ed27af0-ccdd-4036-92e5-ce3cee3771e1)
